x-pack/plugins/security_solution/scripts/endpoint/trusted_apps/index.ts

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
FYI - this utility is to assist with dev. of the Trusted Apps list until we have a `create` flow. I would expect this to be deleted once the create flow is merged.